README.md

```diff
@@ -951,6 +952,41 @@ button on the FMC card).
 [pcie_ethernet]: documentation_source/images/PCIE_ROOT_COMPLEX_ETHERNET.JPG "PCIe root complex with Ethernet PCIe card"
 [pcie_usb]: documentation_source/images/PCIE_ROOT_COMPLEX_USB.JPG "PCIe root complex with USB PCIe card"
 
+## Enabling Video Output ##
+
+Video output on the VCU118 requires additional hardware connected 
+to the both PMOD headers. The recommended setup uses an HDMI cord, 
+a PMOD to DVI board, pullup resistors to 3.3V for each PMOD pin,
+and jumpers to connect the setup to the VCU118's PMOD pins. 
+
+A Vivado project can be generated with this subsystem activated. Doing
+so requires setting `en_frame_buff=1` in `./setup_soc_project`.
+```
+# Parse the processor selection
+proc_picker $1
+
+no_xdma=0
+# Currently there is an issue when no_xdma=1 and en_frame_buff=1 as the block
+# design of the video output was created with no_xmda=1. Setting both to 1
```

Comment:
If there's a problem with `no_xdma=1`, was the block design created with `no_xdma=0`? That'd make the `=1` in line 970 a typo; otherwise, I guess, it's the `no_xdma=1` in line 969 that's a typo. In any event I wonder if we can (in a straightforward way) fix the problem before we merge, though if it's likely to take more than an hour or so it's probably not worth it.

Reply:
(also, the `no_xmda` is definitely a typo, regardless)